lib/genome/updaters/chembl.rb

```diff
@@ -0,0 +1,73 @@
+require 'pg'
+
+module Genome
+  module Updaters
+    class Chembl
+      def self.chembl_db_name
+        'chembl_23'
+      end
+
+      def self.flag(string)
+        h = {
+            '0'=>false,
+            '1'=>true,
+        }
+        return h[string]
+      end
+
+      def self.mol_col_hash
+        @@mol_col_hash ||= DataModel::ChemblMolecule.columns_hash
+      end
+
+      def self.syn_col_hash
+        @@syn_col_hash ||= DataModel::ChemblMoleculeSynonym.columns_hash
+      end
+
+      def self.connection
+        @@connection ||= PG.connect(dbname: chembl_db_name)
+      end
+
+      def self.init_records_from_db # This is for an initial, non-incremental load into the chembl_molecule table
+        new_records = []
+        i = 0
+        connection.exec( "SELECT * FROM molecule_dictionary").each do |record|
+          new_records << get_params(record, mol_col_hash)
+          unless i % 50000
```

Comment:
Everything but `nil` and `false` is `true` in ruby, so unless I'm missing something here; this will be true every single time it runs, regardless of the value of `i`.